static/app/views/onboarding/otherSetup.tsx

```diff
@@ -0,0 +1,160 @@
+import {Organization, Project} from 'app/types';
+import {analytics} from 'app/utils/analytics';
+import getDynamicText from 'app/utils/getDynamicText';
+import withApi from 'app/utils/withApi';
+import withOrganization from 'app/utils/withOrganization';
+import {ProjectKey} from 'app/views/settings/project/projectKeys/types';
+
+import FirstEventFooter from './components/firstEventFooter';
+import SetupIntroduction from './components/setupIntroduction';
+import {StepProps} from './types';
+
+type AnalyticsOpts = {
+  organization: Organization;
+  project: Project | null;
+  platform: PlatformKey | null;
+};
+
+const recordAnalyticsDocsClicked = ({organization, project, platform}: AnalyticsOpts) =>
+  analytics('onboarding_v2.full_docs_clicked', {
+    org_id: organization.id,
+    project: project?.slug,
+    platform,
+  });
+
+type Props = StepProps & {
+  api: Client;
+  organization: Organization;
+} & AsyncComponent['props'];
+
+type State = {
+  keyList: Array<ProjectKey> | null;
+} & AsyncComponent['state'];
+
+class OtherSetup extends AsyncComponent<Props, State> {
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      keyList: null,
+    };
+  }
+
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization, project} = this.props;
+    return [['keyList', `/projects/${organization.slug}/${project?.slug}/keys/`]];
+  }
+
+  handleFullDocsClick = () => {
+    const {organization, project, platform} = this.props;
+    recordAnalyticsDocsClicked({organization, project, platform});
+  };
+
+  render() {
+    const {organization, project} = this.props;
+    const {keyList} = this.state;
+
+    const currentPlatform = 'other';
+
+    const introduction = (
+      <React.Fragment>
+        <SetupIntroduction
+          stepHeaderText={t('Prepare the Other SDK')}
+          platform={currentPlatform}
+        />
+        <motion.p
+          variants={{
+            initial: {opacity: 0},
+            animate: {opacity: 1},
+            exit: {opacity: 0},
+          }}
+        >
+          {tct(
+            "Don't have a relationship with your terminal? [link:Invite your team instead].",
+            {
+              link: (
+                <Button
+                  priority="link"
+                  data-test-id="onboarding-getting-started-invite-members"
+                  onClick={openInviteMembersModal}
+                />
+              ),
+            }
+          )}
+        </motion.p>
+      </React.Fragment>
+    );
+
+    const blurb = (
+      <React.Fragment>
+        <p>
+          {tct(`Prepare the SDK for your language following this [docsLink:guide].`, {
+            docsLink: <ExternalLink href="https://develop.sentry.dev/sdk/overview/" />,
+          })}
+        </p>
+
+        <p>
+          {t('Once your SDK is set up, use the following DSN and send your first event!')}
+        </p>
+
+        <p>
+          {'Here is the DSN : '}
```

Comment:
@roggenkemper You should use a translation function here like `tct`

Reply:
added!

Reply:
added!